.github/workflows/stable-spec-tests.yml

```diff
@@ -8,7 +8,7 @@ on:
   workflow_dispatch:
 
 env:
-  FIXTURES_TAG: "verkle@v0.0.6"
+  FIXTURES_TAG: "verkle@v0.0.7-alpha-1"
```

Comment:
TODO: change to `v0.0.7` after all tests are confirmed and a final tag is released.